VEDA_OS01/tests/test_utils.py

```diff
@@ -29,3 +36,28 @@ def test_video_status_update(self, status, update_val_status):
         # Assert the status and call to edx-val api method.
         self.assertEqual(val_api_client.update_video_status.called, update_val_status)
         self.assertEqual(video.transcript_status, status)
+
+    def test_invalidate_fernet_cached_properties(self):
+        """
+        Tests that fernet field properties are properly invalidated.
+        """
+        def verify_model_field_keys(model, field_name, expected_keys_list):
+            """
+            Verifies cached property keys has expected keys list.
+            """
+            field = model._meta.get_field(field_name)
+            # Verify keys are properly set and fetched.
+            self.assertEqual(field.keys, expected_keys_list)
+            self.assertEqual(settings.FERNET_KEYS, expected_keys_list)
```

Comment:
`verify_model_field_keys` should not assert s`ettings.FERNET_KEYS` as we are already doing L#62

Reply:
I will remove from L#62